src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -736,11 +736,15 @@ export class ChatWidget extends Disposable implements IChatWidget {
 			const defaultAgent = this.chatAgentService.getDefaultAgent(this.location, this.input.currentModeKind);
 			const additionalMessage = defaultAgent?.metadata.additionalWelcomeMessage;
 
+			// TODO@havyaus remove this startup experiment once settled
 			const startupExpValue = startupExpContext.getValue(this.contextKeyService);
+			const configuration = this.configurationService.inspect('workbench.secondarySideBar.defaultVisibility');
+			const expIsActive = configuration.defaultValue !== 'hidden';
 			let welcomeContent: IChatViewWelcomeContent;
 			if (startupExpValue === StartupExperimentGroup.MaximizedChat
 				|| startupExpValue === StartupExperimentGroup.SplitEmptyEditorChat
-				|| startupExpValue === StartupExperimentGroup.SplitWelcomeChat) {
+				|| startupExpValue === StartupExperimentGroup.SplitWelcomeChat
+				|| expIsActive) {
```

Comment:
@bhavyaus but under what condition are we removing this treatment again? Or are you planning to always show this welcome treatment even if chat was used?

Reply:
I can reuse the same condition defined here the setup action?
https://github.com/microsoft/vscode/blob/22ee5de7762e9a81c1842f1acff6049126e313fe/src/vs/workbench/contrib/chat/browser/chatSetup.ts#L894-L898